### DISCUSSION TITLE
Recharts Default Props

### DISCUSSION THREAD

**Discussion by nonsookongwu** [recharts/recharts#4709] [Q&A] [answered]
I get the following warning when i use recharts: Warning: XAxis: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.

please when will the next major release be done and  how do we manage this

**Comment by ckifer** [comment, marked as answer]
https://github.com/recharts/recharts/issues/3615#issuecomment-2135955727

This is taken care of. Please upgrade to the latest alpha version (or wait until 2.13 is released). Thanks!
